fe/fe-core/src/main/java/com/starrocks/load/routineload/KafkaTaskInfo.java

```diff
@@ -192,9 +201,11 @@ protected String getTaskDataSourceProperties() {
     }
 
     private TExecPlanFragmentParams plan(RoutineLoadJob routineLoadJob) throws UserException {
+        String label =
+                Joiner.on("-").join(routineLoadJob.getName(), routineLoadJob.getId(), DebugUtil.printId(id), txnId);
```

Comment:
the label have already been defined in the up `createRoutineLoadTask` method. I would recommend to reuse that variable

Reply:
ok